openapi/nexus.json

```diff
@@ -12081,11 +12085,10 @@
               },
               "size": {
                 "description": "total size of the Disk in bytes",
-                "allOf": [
-                  {
-                    "$ref": "#/components/schemas/ByteCount"
-                  }
-                ]
+                "type": "integer",
+                "format": "uint64",
+                "minimum": 1073741824,
+                "maximum": 1098437885952
```

Comment:
Cool to see this in the API, I hope this can make our clients better!

Reply:
So, this actually isn't right. It's not the min/max that's wrong, it's the schema. We still want `ByteCount` to be preserved but we _also_ want the limits. It took me a little digging to figure out what to do here. 

[According to the spec](https://spec.openapis.org/oas/v3.0.3#composition-and-inheritance-polymorphism) two models can be composed into a single object via `allOf`. So what we actually want here is an `allOf` with the `ByteCount` and these limited definitions. My latest commit has something that _should_ produce that, but typify pukes on it here:

https://github.com/oxidecomputer/typify/blob/1f97f167923f001818d461b1286f8a5242abf8b1/typify-impl/src/merge.rs#L690

I plan to look at this more tomorrow.

Reply:
I mean... the `unimplemented!` isn't wrong...